scripts/percona-release.sh

```diff
@@ -9,9 +9,64 @@ if [[ $(id -u) -gt 0 ]]; then
   exit 1
 fi
 #
+
+PRODUCTS_ABBREVIATIONS=("pdmdb" "pdps" "pdpxc" "pmm" "ppg11" "ppg12" "ppg13" "ppg14" "psmdb" "px")
+
+function sort_array {
+  ARRAY=$1
+  local flagforabbr="pdmdb"
+  PRODUCTS=()
+  OTHER_CURRENT_ALIASES=()
+  for element in ${ARRAY[@]}
+    do
+      counter=1
+      for abbr in ${PRODUCTS_ABBREVIATIONS[@]}
+        do
+	  if ! [[ "${element//-/}" =~ ^$abbr.*$ ]]; then
```

Comment:
inverted if. is it possible to rework it?

Reply:
It has been inverted